scrubber_test.go

```diff
@@ -43,7 +44,7 @@ func init() {
 		return
 	}
 	if err := updateCorpus(os.Stderr); err != nil {
-		fmt.Fprintln(os.Stderr, "failed to update test corpus:", err)
+		log.Fatalf("failed to update test corpus: %v", err)
```

Comment:
This will exit the runtime if the corpus cannot update.  So the tests will not run if, for example, you do not have internet connectivity.  I'm not sure this is desirable... but I do see the idea in making sure the corpus reflects the most up-to-date version.  Is this what you'd prefer?  Alternatively, we can make sure that CI is always running against the latest corpus and keep local tests updating the corpus on a best-effort basis.

Reply:
:shrug: yeah my priority is that CI provides accurate results. Couldn't someone just use the short flag if they don't have internet connectivity?

Reply:
Yeah, but then they couldn't test against the corpus (and they likely have the latest one locally anyway).  They could always come in and change this line though.  And it's likely not a common scenario, so I'm fine with whatever you prefer.

Reply:
I would panic here though over `log.Fatal`.  That way the person running the tests gets the line number and can figure out how to "fix" things.

Reply:
ah ok - I'm so confused about go's fatal exit strategies...

Reply:
Heh heh.  I don't blame you.  `log.Fatal` is like calling `log.Print`, then `os.Exit(1)`.  I think the main advantage of the log package is timestamps, but that's not too important here.